lib/active_model/serializer/concerns/associations.rb

```diff
@@ -85,18 +85,26 @@ def associate(reflection)
       #
       def associations(include_directive = ActiveModelSerializers.default_include_directive, include_slice = nil)
```

Comment:
This method is called in `3` places in the lib, and I think it should be called once.  The serializer should yield the association serialized per include/field options to the adapter for processing...